WordPress/Classes/Services/AuthenticationService.swift

```diff
@@ -84,7 +95,16 @@ class AuthenticationService {
 
                 self.getAuthCookiesForWPCom(username: username, authToken: authToken, success: { cookies in
                     cookieJar.setCookies(cookies) {
-                        success()
+
+                        // Verify that the WP.com auth cookie was indeed returned by the server.
```

Comment:
I'd consider removing this comment as it's not adding any information to the code that's right below it.

It's not a big deal, so feel free to ignore it.  But I always try to avoid comments that explain what the code does since the value they provide is low (since any developer should be able to tell what the code does), vs the added effort of maintaining it over time as the code changes.  My 2 cents!

Reply:
That's understandable, I'll go ahead and remove it.